gulp/tasks/styles.js

```diff
@@ -2,22 +2,25 @@ var gulp            = require('gulp'),
     autoprefixer    = require('gulp-autoprefixer'),
     browserSync     = require('browser-sync'),
     changed         = require('gulp-changed'),
-    config          = require('../config').styles,
+    config          = require('../config'),
```

Comment:
When I was pulling in the "bower component" for `afontgarde` I needed to pull the CSS from two places, and so I chose to define them each individually. Now that this is rolled into the main "styles" folder, this is unnecessary.